test/paths/include-paths-filter-myltiple-reference-targets/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cat input.json | swagger-json-filter --include-paths="\/estimates*"
```

Comment:
Why do you put the asterisk at the end of the regex? It says that `e` letter can be replaced many times. Why is it useful? I suggest to remove the asterisk or put `.*` instead (`\/estimates.*`) to allow any characters after the `/estimates` url fragment.

**This should be corrected in all tests**